script.js

```diff
@@ -5,7 +5,8 @@ const {
 } = require('url');
 const fs = require('fs');
 const { transpose } = require('matrix-transpose');
-const median = require('median')
+const median = require('median');
+const percentile = require('percentile');
```

Comment:
this dependency needs to be added to the package.json so it's updated for other people running the script

you can do that automatically by using the --save (-s shorthand) when you install a package, such as
`npm install --save percentile`